From 03797d28ce9b78d74b059c7e68f077311c774d77 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Thu, 11 Dec 2025 15:13:36 +0000
Subject: [PATCH 3/3] Fix an unsupported escape sequence in a string literal in
 a comment

---
 fedora/client/fas2.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fedora/client/fas2.py b/fedora/client/fas2.py
index c7b5119..dd5366b 100644
--- a/fedora/client/fas2.py
+++ b/fedora/client/fas2.py
@@ -391,8 +391,8 @@ class AccountSystem(BaseClient):
         The people are all approved in the group.  Unapproved people are not
         shown.  The format of data is::
 
-            \[{'username': 'person1', 'role_type': 'user'},
-            \{'username': 'person2', 'role_type': 'sponsor'}]
+            [{'username': 'person1', 'role_type': 'user'},
+            {'username': 'person2', 'role_type': 'sponsor'}]
 
         role_type can be one of 'user', 'sponsor', or 'administrator'.
 
-- 
2.52.0

